### PR TITLE
Enforce braces for code-blocks

### PR DESCRIPTION
**Pull request by pranavkm**
Contributes to https://github.com/dotnet/aspnetcore/issues/24055

### CODE REVIEW

File: src/Identity/test/Shared/MockHelpers.cs [hunk 1]
```diff
@@ -67,9 +67,13 @@ public static ILookupNormalizer MockLookupNormalizer()
         var normalizerFunc = new Func<string, string>(i =>
         {
             if (i == null)
```
Comment (TanayParikh) [inline src/Identity/test/Shared/MockHelpers.cs:69]: Unrelated to this PR, but should we consider enforcing `is null` instead of `== null`.

**Review by TanayParikh — APPROVED** [review]
💯

File: src/Identity/test/Shared/MockHelpers.cs (hunk 1 above)
Comment (pranavkm) [reply src/Identity/test/Shared/MockHelpers.cs:69]: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0041. I think we should bring it up as part of a DOI thing first though.

Comment (TanayParikh) [reply src/Identity/test/Shared/MockHelpers.cs:69]: https://github.com/dotnet/aspnetcore/issues/39323

**Comment by TanayParikh** [comment]
Tests failing due to infra 503. Re-triggered.

**Comment by ylabade** [comment]
@pranavkm I would like to know why you put `csharp_prefer_braces = true` + `dotnet_diagnostic.IDE0011.severity = warning` ? instead of only `csharp_prefer_braces = true:warning` ? 
Is `:warning` a bad practice ? Or is it just a convention compared to the rest of the rules in the editorconfig ?

**Comment by pranavkm** [comment]
@ylabade it apparently doesn't work for CLI builds. See https://github.com/dotnet/roslyn/issues/44201

**Pull request merged**